pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -5,13 +5,12 @@
 
 """
 
-import logging as log
 import warnings
 
 import numpy as np
 
 from .base import Callback
-
+from ..core import root_logger
```

Comment:
use absolute imports
```suggestion
from pytorch_lightning.core import root_logger
```